uReplicator-Controller/src/main/java/com/uber/stream/kafka/mirrormaker/controller/core/IdealStateBuilder.java

```diff
@@ -120,7 +143,7 @@ public static IdealState expandCustomRebalanceModeIdealStateFor(IdealState oldId
         customModeIdealStateBuilder.assignInstanceAndState(Integer.toString(i),
             liveInstance.getInstanceName(),
             "ONLINE");
-        liveInstance.addTopicPartition(new TopicPartition(topicName, i));
```

Comment:
I don't really understand the logic here, why add only one topic partition to the liveinstance ?